src/discussions/data/api.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
Could the new `discussions/data` directory be restructured as `discussions/course-config/data` (or something similar) for 2 main reasons:
1. Ease of understanding for future developers
2. Maintaining consistency with other sub-directories in `discussions` directory

Reply:
I see the structure as follows:
- /data for non-discussions related API and data
- /discusions/data for data that's relevant in multiple places
- /discussions/___/data for data specific to a particular component

I've kept this here since it's more global. We don't have any course-config related components or UI. It's used by other UI.